vmaas_sync/metrics_cyndi.go

```diff
@@ -0,0 +1,165 @@
+package vmaas_sync //nolint:golint,stylecheck
+
+import (
+	"app/base/database"
+	"app/base/models"
+	"app/base/utils"
+	"encoding/json"
+	"fmt"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"reflect"
+	"time"
+)
+
+var (
+	cyndiSystemsCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_systems",
+	}, []string{"type"})
+
+	cyndiTagsCnt = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "vmaas_sync",
+		Name:      "cyndi_tags_count",
+	}, []string{"type"})
+)
+
+type InventoryHosts []struct {
+	DisplayName   string
+	Tags          string
+	SystemProfile string
+}
+
+type Tags []struct {
```

Comment:
It'd be much nicer to have a `Tag` struct instead of `Tags` and use slice elsewhere. Also these data- definition structs are usually stored in `base/models.go` so that we can reuse them for other purposes.